sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -1661,6 +1661,11 @@ private static String unquote(String value) {
             .replace("''", "'");
     }
 
+    private static String unquoteEscaped(String value) {
```

Comment:
could we unify this with `unquote` by creating a method `unquote(String value, int beginIndex)` instead?

Reply:
I though about it but did not do in the end as it's a 2 line method and on similar cases people mentioned that the benefit does not justify the added complexity. But since you both prefer that I'll go on with it.